analysis/baconbits-plot.ipynb

```diff
@@ -9,13 +9,14 @@
     "from __future__ import print_function, division\n",
     "from collections import defaultdict, OrderedDict\n",
     "import gzip\n",
-    "import pickle\n",
+    "import lz4.frame as lz4f\n",
+    "import cloudpickle as cpkl\n",
     "import json\n",
     "import re\n",
     "import os\n",
     "\n",
     "import uproot\n",
-    "%matplotlib nbagg\n",
+    "%matplotlib inline\n",
```

Comment:
No `nbagg` at vanderbilt?

Reply:
Interface gets all wonky, imho. Feel free to give it a try.